plugins/orchestrator/src/components/OrchestratorPage/OrchestratorPage.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useMemo } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useAsync } from 'react-use';
+
+import {
+  ContentHeader,
+  Progress,
+  ResponseErrorPanel,
+} from '@backstage/core-components';
+import { useApi, useRouteRef } from '@backstage/core-plugin-api';
+
+import { Button, Grid } from '@material-ui/core';
+
+import {
+  workflow_title,
+  WorkflowItem,
+} from '@janus-idp/backstage-plugin-orchestrator-common';
+
+import { orchestratorApiRef } from '../../api';
+import { newWorkflowRef, workflowInstancesRouteRef } from '../../routes';
+import { BaseOrchestratorPage } from '../BaseOrchestratorPage/BaseOrchestratorPage';
+import { OrchestratorSupportButton } from '../OrchestratorSupportButton/OrchestratorSupportButton';
+import { WorkflowsTable } from '../WorkflowDefinitionsListComponent/WorkflowDefinitionsListComponent';
+
+export const OrchestratorPage = () => {
+  const orchestratorApi = useApi(orchestratorApiRef);
+  const navigate = useNavigate();
+  const newWorkflowLink = useRouteRef(newWorkflowRef);
+  const instancesLink = useRouteRef(workflowInstancesRouteRef);
+
+  const { value, error, loading } = useAsync(async (): Promise<
+    WorkflowItem[]
+  > => {
+    const data = await orchestratorApi.listWorkflows();
+    return data.items;
+  }, []);
+
+  const isReady = useMemo(() => !loading && !error, [loading, error]);
```

Comment:
There is no benefit in the `useMemo()` here, just disdvantages

Reply:
There is a refactoring going on and this component will likely be replaced with new code.